src/api/spec/models/project_spec.rb

```diff
@@ -425,4 +426,51 @@
       end
     end
   end
+
+  describe '#prepend_kiwi_config' do
+    let!(:project) { create(:project, name: 'backend_project') }
+    let(:method) { project.prepend_kiwi_config }
+    let(:config_url) { "http://localhost:3200/source/#{project.name}/_config" }
+
+    before do
+      CONFIG['global_write_through'] = true
```

Comment:
You should let this CONFIG as it was before... so add an `after` block to reset the value